include/benchmark/benchmark_api.h

```diff
@@ -709,8 +709,12 @@ class Fixture: public internal::Benchmark {
       this->TearDown(st);
     }
 
+    // These will be deprecated ...
     virtual void SetUp(const State&) {}
     virtual void TearDown(const State&) {}
+    // ... In favor of these.
+    virtual void SetUp(State& st) { SetUp(static_cast<State const&>(st)); }
```

Comment:
Please be consistent wrt the placement of `const`.  This library generally puts it before the type name.  (I am in the "after the type name" camp, but consistency trumps.)